packages/widgets/src/MultiSelect/Menu.js

```diff
@@ -15,7 +15,6 @@ const onDisabledClick = (_, e) => {
 
 const createHandler = ({ isActive, onChange, selected, value }) => (_, e) => {
     e.stopPropagation()
-    e.preventDefault()
```

Comment:
This line was removed to fix the following bug:
- Clicking an option Checkbox once worked correctly
- But after that first click it would always require two clicks to toggle an option Checkbox

After consulting @ismay, a test was added to verify the bug is gone, see https://github.com/dhis2/ui/pull/10/files#r408372824

Reply:
I'd still like to look into why this was going wrong exactly. I wonder also how many of the stopPropagation's and preventDefault's we could get rid of btw, but that's maybe a different issue.